packages/frontend/core/src/components/root-app-sidebar/index.tsx

```diff
@@ -158,6 +154,7 @@ export const RootAppSidebar = (): ReactElement => {
           <QuickSearchInput
             className={quickSearch}
             data-testid="slider-bar-quick-search-button"
+            data-event-props="$.navigationPanel.generalFunction.quickSearch"
```

Comment:
The original definition is to track when opening. It seems that every mouse click will track here?

Reply:
what do you mean by every click?

Reply:
sorry, I misunderstood